src/styles/card.css

```diff
@@ -0,0 +1,115 @@
+@import "./stars.css";
+
+.textSmall {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.textMedium {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.textLarge {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card {
+  flex-direction: column;
+
+  display: flex;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__image {
+  align-self: center;
+
+  display: flex;
+
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+}
+
+.card__pcode {
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
+
+  display: flex;
+
+  margin-bottom: 24px;
+}
+
+.card__review {
+  margin: 0;
+}
+
+.card__price {
+  justify-content: space-between;
+
+  display: flex;
+
+  margin-bottom: 16px;
+}
+
+.card__price-tag {
+  margin: 0;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price-figure {
+  margin: 0;
+}
```

Comment:
You can reset margin: 0 in your index.css on div and other required tags to avoid repetition of code